waggle-dance-core/pom.xml

```diff
@@ -97,6 +97,11 @@
       <artifactId>micrometer-registry-graphite</artifactId>
     </dependency>
 
+    <dependency>
```

Comment:
To be really pedantic this should go below the jmx one if we want to keep these alpha-sorted by groupid.artifactid where possible.